Gopkg.toml

```diff
@@ -11,11 +11,9 @@
   name = "github.com/uber/jaeger-lib"
   version = ">=v1.5.0"
 
-# Pin to master branch until there is a more recent stable release:
-#   https://github.com/prometheus/client_golang/issues/375
 [[override]]
   name = "github.com/prometheus/client_golang"
-  branch = "master"
+  version = "v0.9.2"
```

Comment:
Please add a comment giving the reason we need an override, any background why this specific version, and the conditions under which the override can be removed.

Reply:
Well, there was comment before (TODO: Pin this version), so I just fullfillted TODO, but sure, can add more info.

Reply:
Where do you see "TODO"?  I see "Pin to master branch until there is a more recent stable release"
Can the override be removed entirely now?

Reply:
? Isn't that TODO-like comment? Didn't I do exactly this? :confused: 

> "Pin to master branch until there is a more recent stable release"

Anyway, override is required due to jaeger contstraint. Commented all and added in another PR: 
https://github.com/cortexproject/cortex/pull/1353

One it will be merged, I will remove this change from this PR.